core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/AmbariClusterService.java

```diff
@@ -128,11 +128,14 @@ public Cluster retrieveClusterForCurrentUser(Long stackId) {
     @Override
     public String getClusterJson(String ambariIp, Long stackId) {
         Stack stack = stackRepository.findOne(stackId);
+        if (stack.getAmbariIp() == null) {
+            throw new NotFoundException(String.format("Ambari IP is not set for stack.[id: %s]", stackId));
```

Comment:
This message could be "Ambari server could not be available for the stack......" mabye that is more user friendly :)